ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml

```diff
@@ -10,7 +10,7 @@ ethernet_interfaces:
     mtu: 1500
     l2_mtu: 8000
     l2_mru: 8000
-    type: routed
+    type: switched
```

Comment:
We should not need this to be set with the new model.

Reply:
fixed